File: Practical_Task_2/Wrong_Ones/Java/BankAccount.java
public class BankAccount {
    public void calcInt(int bal, double rate, int time) {
        double intr = bal * rate * time;
        if (intr > 1000) {
            System.out.println("High interest");
        } else {
            System.out.println("Low interest");
            return;
        }
        System.out.println("Error: Calculation issue.");
    }

    public static void main(String[] args) {
        BankAccount account = new BankAccount();
        account.calcInt(50000, 0.05, 5);
    }
    
    // Intentional compile-time error: Incorrect data type
    public void incorrectType() {
        int invalidType = "This is a string";  // This will cause a compile-time error
    }
}

File: Practical_Task_2/Wrong_Ones/BankAccount.php
<?php

class BankAccount {
    public function calcInt($bal, $rate, $time) {
        $intr = $bal * $rate * $time;
        if ($intr > 1000) {
            echo "High interest\n";
        } else {
            echo "Low interest\n";
            return;
        }
        echo "Error: Calculation issue.\n";
    }

    public function incorrectType() {
        // Intentional error: Trying to assign a string to an integer variable
        $invalidType = (int) "This is a string";  // Type casting string to integer
    }
}

$account = new BankAccount();
$account->calcInt(50000, 0.05, 5);

// Uncomment the following line to trigger the intentional error
// $account->incorrectType();